### PR TITLE
React Native (set up) & Git (commit & pr) Done

### PR DESCRIPTION
**Pull request by Vincentzs**
Added 
-React Native Expo set up
-Useful git commands (stash, commit)
-Update PR

### CODE REVIEW

File: Topics/Development_Process.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+        #Note: NEVER remove the "--soft" tag, because if you remove it. The reset will be a hard reset and will remove all the changed files from the commit and sometimes it will lead into lost files and you will need to go back to "1. Retrieve Lost Commits". So for a safe practice, just use the soft reset ("--soft")
+        Link: https://stackoverflow.com/questions/12481639/remove-file-from-latest-commit
+
+4. Pull-requests
+    1. Update Pull-requests
+        After pushing your commits to create a new pull request, sometimes you realize you need to change a file that is already committed on the pull request. So you would change the file, git add, commit, push it and then realize you cannot push it. The following command will help you resolve this issue.
+
+        git push origin master --force
+
+        #Note: Use with precaution because we are using the force push. So try to limit the usage of this to directly push codes to the master branch.
```
Comment (amir-alleyne) [inline Topics/Development_Process.md:37]: Good job leaving a warning about using --force. It is often very dangerous to force commits and git actions when working in a team as you may possibly overwrite teammates work unexpectedly and cause unintended behaviour.

File: Topics/Tech_Stacks.md [hunk 2]
```diff
@@ -1 +1,25 @@
...
+            1. Install expo
+                npm install -g expo-cli
+            2. Initialize expo
+                expo init your_app_name
+                #Note: you will be able to choose "blank", "blank (TypesScript)", "tabs (TypeScript)", or "minimal". I personally prefer blank because often times I need to delete files I don't want. 
+            3. Start expo
+                cd your_app_name
+                expo start
+        3. Run expo on your phone or computer
+            1. Phone
```
Comment (amir-alleyne) [inline Topics/Tech_Stacks.md:21]: It is also possible to run an emulator on the Xcode app on laptops. This way you can see live changes without using a cell phone.

File: Topics/Development_Process.md (hunk 1 above)
Comment (lmg8) [reply Topics/Development_Process.md:37]: Just to add to this, might be worth mentioning a safer alternative to --force which is --force-with-lease.

File: Topics/Tech_Stacks.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+        2. Command
+            1. Install expo
+                npm install -g expo-cli
+            2. Initialize expo
+                expo init your_app_name
+                #Note: you will be able to choose "blank", "blank (TypesScript)", "tabs (TypeScript)", or "minimal". I personally prefer blank because often times I need to delete files I don't want. 
+            3. Start expo
+                cd your_app_name
+                expo start
+        3. Running expo
```
Comment (lmg8) [inline Topics/Tech_Stacks.md:20]: Is it possible for you to format the inline code similar to how [MySQL page](https://learning-software-engineering.github.io/Topics/Tech_Stacks/Learning_MySQL.html) was done?

It's hard to tell what code we should run on first look:
image

File: Topics/Tech_Stacks.md [hunk 4]
```diff
@@ -1 +1,28 @@
 ## Tech Stacks
+1. React Native
+    1. Set up
+        1. Installation
+            1. Install Node.js and npm (Node Package Manager) on your computer. You can download and install Node.js from the official website: https://nodejs.org/en/
+                #Note: Suggested to download the LTS version because it is generally more stable than the Current or Latest version.
+            2. Install expo go on your ios device (https://apps.apple.com/us/app/expo-go/id982107779) or android device (https://play.google.com/store/apps/details?id=host.exp.exponent&hl=en_CA&gl=US&pli=1)
+            3. Download VS Code Extensions
+                1. ES7+ React/Redux/React-Native snippets (Extension ID: dsznajder.es7-react-js-snippets)
+                2. React Native Tools (Extension ID: msjsdiag.vscode-react-native)
```
Comment (lmg8) [inline Topics/Tech_Stacks.md:10]: Curious, what are these extensions for? Are there similar extensions for users who do not use VS Code?

**Review by lmg8 — CHANGES_REQUESTED** [review]
May you move the React Native details in it's own file under [tech_stacks](https://github.com/learning-software-engineering/learning-software-engineering.github.io/tree/main/Topics/Tech_Stacks) and then link it inside [Tech_Stacks.md](https://github.com/learning-software-engineering/learning-software-engineering.github.io/blob/main/Topics/Tech_Stacks.md)?

Similarly, for Git, there seems to already be a folder under software_tools:
https://github.com/learning-software-engineering/learning-software-engineering.github.io/tree/main/Topics/Software_Tools/Git
Although, I'm not entirely sure whether it should be under software tools or development process.

**Pull request closed without merging**